package.json

```diff
@@ -5,23 +5,25 @@
   "license": "MIT",
   "repository": "https://github.com/asciidoctor/gulp-asciidoctor",
   "author": {
-    "name": "John Dong",
-    "email": "dongwqs@sina.com",
-    "url": "https://github.com/dongwq/gulp-asciidoctor"
+    "name": "Armin Pfarr",
```

Comment:
I think you should add multiples authors:

```json
"authors": [
  "John Dong <dongwqs@sina.com>",
  "Armin Pfarr <arp@henriette-einstein.com>"
]
```

The original author seems to be https://github.com/jeremy889988 but he was never included in the package.json

Reply:
Sorry, I did not want to offend anybody here. I'll correct it immediately!

Reply:
No worries :smile: